tests/ui_tests/reporting/report_master_diff.py

```diff
@@ -0,0 +1,191 @@
+import html
+import shutil
+import tempfile
+from pathlib import Path
+
+import dominate
+from dominate.tags import br, h1, h2, hr, i, p, table, td, th, tr
+
+import download
+
+REPORTS_PATH = Path(__file__).parent.resolve() / "reports" / "master_diff"
+RECORDED_SCREENS_PATH = Path(__file__).parent.parent.resolve() / "screens"
+
+
+def get_diff():
+    master = download.fetch_fixtures_master()
+    current = download.fetch_fixtures_current()
+
+    # removed items
+    removed = {test: master[test] for test in (master.keys() - current.keys())}
+    # added items
+    added = {test: current[test] for test in (current.keys() - master.keys())}
+    # items in both branches
+    same = master.items() - removed.items() - added.items()
+    # create the diff
+    diff = dict()
+    for master_test, master_hash in same:
+        if current.get(master_test) == master_hash:
+            continue
+        diff[master_test] = {
+            "master": master[master_test],
```

Comment:
You mean to have `diff[master_test] = (master[master_test], current[master_test])`? Yeah it could but this seemed a bit more explicit and readable, ins't it?

Reply:
maybe, but dict is not supposed to be a poor man's object. use namedtuple if you want to be extra explicit, but that doesn't seem worth it here

Reply:
True. I'll change it to tuple.